game.py

```diff
@@ -137,8 +142,34 @@ def pvp_round(self) -> None:
             sleep(1)
             self.arena.pick_augment()
             sleep(2.5)
-        if self.round in ("2-1", "2-5"):
+        if self.round in ("2-1"):
+            """Level to 4 at 2-1"""
             self.arena.buy_xp_round()
+            print(f"\n[LEVEL UP] Lvl. {arena_functions.get_level()}")
+        if self.round in ("2-5"):
+            """Level to 5 at 2-5"""
+            while arena_functions.get_level() < 5:
+                self.arena.buy_xp_round()
+            else:
+                print(f"\n[LEVEL UP] Lvl. {arena_functions.get_level()}")
```

Comment:
Don't need any else's here
```suggestion
            print(f"\n[LEVEL UP] Lvl. {arena_functions.get_level()}")
```